### DISCUSSION TITLE
EcoLink PIR stuck at ProtocolInfo

### DISCUSSION THREAD

**Discussion by jchonig** [zwave-js/zwave-js#6955] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

When adding EcoLink PIR nodes they are stuck in ProtocolInfo (i.e. they do not interview).

I started by moving one from another home automation platform.  I get the same behavior with a shiny new PIR out of the box.

Manufacturer, Product and Product code all show up as 0xXXXX

What did you expect to happen instead?

Interview process to complete and Z-Wave JS UI and HA to see the device mfr and model, ...

 
1. Add device via Home Assistant Z-Wave device.

or

1. Add device via Z-Wave JS UI Inclusion
2. Enter device name and location
3. Leave default security settings
4. Press button on back of device (or remove battery for 5 seconds)
5. Device is added w/o security

Anything else we should know?

I have 3 of these working, but they were added before I switched Home Assistant to Z-Wave JS UI.

### Software versions

Driver (node-zwave-js): ...

zwave-js: 12.11.1

Z-Wave JS UI: ...

zwave-js-ui: 9.14.1

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

Current version: 3.8.1

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: EcoLink
Model name: .PIR-ZWAVE2.5-ECHO
Node ID: 86 and then 87

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2024-06-23.log](https://github.com/user-attachments/files/15946375/z-ui_2024-06-23.log)

[a comment by an automated account is not shown]

**Comment by jchonig** [comment]


**Reply by AlCalzone** [reply]
Please read the link posted by the bot again. None of the files you posted are the driver log.

**Comment by jchonig** [comment]
This time the driver log.  Node 087 for exclusion and Node 088 for inclusion.  Followed by a re-interview.

[zwavejs_2024-06-24.log](https://github.com/user-attachments/files/15956191/zwavejs_2024-06-24.log)

**Reply by AlCalzone** [reply]
I think you might be affected by the bug fixed in https://github.com/zwave-js/node-zwave-js/releases/tag/v12.11.2

**Reply by jchonig** [reply, marked as answer]
I just updated HA to the latest Z-Wave JS integration and now inclusion works correct.

Thanks for the help!
